### PR TITLE
refactor(typescript-estree): correct warnings reported by sonarlint

### PR DESCRIPTION
**Pull request by armano2**
This PR fixes issues found in code by static code analysis by SonarLint.

### CODE REVIEW

File: packages/typescript-estree/src/ast-converter.ts [hunk 1]
```diff
@@ -11,11 +11,11 @@ import { convertTokens } from './node-utils';
 import ts from 'typescript';
 import { Extra } from './temp-types-based-on-js-source';
 
-export default (
+export default function astConverter (
```
Comment (armano2) [inline packages/typescript-estree/src/ast-converter.ts]: its easier to profile named functions

**Comment by armano2** [comment]
Codecov don't like my change (even if it's just refactoring)

**Comment by JamesHenry** [comment]
Not sure if I'm a huge fan of solving this by lowering coverage requirements. At a quick glance it seems like you can resolve this by adding a test for the case where code is anything other than a string?

If we have that behaviour in there it seems reasonable we should test for it?

**Comment by JamesHenry** [comment]
@armano2 loggerFn is needed - prettier and some others use it

**Comment by armano2** [comment]
ohh, i used bad wording, there is no tests if it's actually working

**Pull request merged**